Marlin/src/inc/SanityCheck.h

```diff
@@ -3568,7 +3568,7 @@ static_assert(_PLUS_TEST(3), "DEFAULT_MAX_ACCELERATION values must be positive."
   #error "LED_CONTROL_MENU requires an LCD controller that implements the menu."
 #endif
 
-#if ENABLED(CUSTOM_MENU_MAIN) && NONE(HAS_MARLINUI_MENU, TOUCH_UI_FTDI_EVE, TFT_LVGL_UI)
+#if ENABLED(CUSTOM_MENU_MAIN) && NONE(HAS_MARLINUI_MENU, TOUCH_UI_FTDI_EVE, TFT_LVGL_UI, EXTENSIBLE_UI)
   #error "CUSTOM_MENU_MAIN requires an LCD controller that implements the menu."
```

Comment:
`CUSTOM_MENU_MAIN` can't possibly be something implemented as part of ExtUI. It depends on the LCD implementation to create the menu items in its own distinct way.

Reply:
Lulzbot currently loops over the list and draws each element that exists in a similar implementation to the 12864 display. I can link their repo if needed for an existing implementation using it if needed.

Reply:
Yeah, it would be great to add that as a standard part of ExtUI. That would be a very nice solution to the issue.